### PR TITLE
adding web console blurbs

### PR DESCRIPTION
**Pull request by nalhadef**
[REPLACED BY PR41381](https://github.com/openshift/openshift-docs/pull/41381)

Ready for Peer Review (Feb 2)

For Main/4.10

Jira: [https://issues.redhat.com/browse/OSDOCS-2657](https://issues.redhat.com/browse/OSDOCS-2657)

New content: adding Note about the web console to Viewing a network policy, Editing a network policy, and Deleting a network policy.

Direct link to changes:

- [Viewing
](https://deploy-preview-41161--osdocs.netlify.app/openshift-enterprise/latest/networking/network_policy/viewing-network-policy.html)
- [Editing
](https://deploy-preview-41161--osdocs.netlify.app/openshift-enterprise/latest/networking/network_policy/editing-network-policy.html)
- [Deleting
](https://deploy-preview-41161--osdocs.netlify.app/openshift-enterprise/latest/networking/network_policy/deleting-network-policy.html)

- [Creating*](https://deploy-preview-41161--osdocs.netlify.app/openshift-enterprise/latest/networking/network_policy/creating-network-policy.html)

*A similar note for creating a policy was previously added. With this new wording, I've updated that note to match.

Ready for QE and SME review:
QE: @memodi
SME: @jotak

### CODE REVIEW

**Comment by jotak** [comment]
hey @nalhadef , the form view only allows to create network policies, not to update or delete them.
Updates and Deletes are done from the list or details views, via the Actions dropdown:

- List:
Capture d’écran de 2022-01-31 12-50-54

- Details:
Capture d’écran de 2022-01-31 12-50-32

**Review by jotak — APPROVED** [review]

File: modules/nw-networkpolicy-delete.adoc [hunk 1]
```diff
@@ -75,3 +76,10 @@ ifdef::multi[]
 endif::multi[]
 :!name:
 :!role:
+
+[NOTE]
+====
+If you log in with a user with the `cluster-admin` role in the console, then you
```
Comment (sagidlow) [inline modules/nw-networkpolicy-delete.adoc:82]:
```suggestion
If you log in with `cluster-admin` permissions in the web console, then you
```

Just a suggestion for easier reading. I would apply this change to the other NOTE below.

File: modules/nw-networkpolicy-delete.adoc [hunk 2]
```diff
@@ -75,3 +76,10 @@ ifdef::multi[]
...
+have a choice of deleting a network policy in any namespace in the cluster
+directly from the YAML view or from the policy in the web console through the *Actions* menu.
```
Comment (sagidlow) [inline modules/nw-networkpolicy-delete.adoc:84]:
```suggestion
directly from the YAML file or from the policy in the web console through the *Actions* menu.
```

YAML file? If you make this change, make the same to the other NOTES below.

**Review by sagidlow — COMMENTED** [review]
Couple of nits. :)

**Pull request closed without merging**